config/errros.go

```diff
@@ -0,0 +1,8 @@
+package config
+
+import "fmt"
+
+var (
+	ErrStrictModeValidation       = fmt.Errorf("failed strict mode check")
```

Comment:
I know this was in the original file https://github.com/lyft/flytestdlib/blob/master/config/errors.go#L6 but is it intentional that we don't want to use the `ErrorCode` class that flytestdlib defines?

Reply:
no we want the error specifically to compare i think